pallets/capacity/src/tests/unstaking_tests.rs

```diff
@@ -102,8 +102,8 @@ fn unstake_errors_max_unlocking_chunks_exceeded() {
 	new_test_ext().execute_with(|| {
 		let token_account = 200;
 		let target: MessageSourceId = 1;
-		let staking_amount = 10;
-		let unstaking_amount = 1;
+		let staking_amount = 60;
+		let unstaking_amount = 10; // TODO: we need to figure out how to deal with fractional capacity.
```

Comment:
Re: TODO. I thought Capacity was a u128 like Balances are. In that case the "dust" can just be floored I think.